.nvmrc

```diff
@@ -1 +1 @@
-8.4.0
+8.9.4
```

Comment:
+1, we should update the rest of the terra projects to 8.9.4. It was [mentioned that webpack 4 will have the best performance on node >= 8.9.4](https://twitter.com/wSokra/status/967852475918274561).